### ISSUE TITLE
Original paper results

### ISSUE THREAD

**Issue by marcociccone** [thomasj02/nfsp-pytorch#1]
Hi @thomasj02, 
Thank you for your pytorch implementation! I'm spending some time trying to reproduce the results from the original paper and the lua code, but unfortunately with no luck.

Have you been able to reach good exploitability values? The paper says around 0.06 for Leduc Poker and indeed I can reach ~0.08 with the lua code. With your code instead I'm able to get ~0.6 with 800K iterations. What is your experience?

Thanks, 
Marco

**Comment by thomasj02** [comment]
I've mostly put this aside, as NFSP has been largely superseded by the more recent Deep CFR, Single Deep CFR, and Pluribus papers.

**Comment by marcociccone** [comment]
yeah, I'm well aware of those papers. I need NFSP as baseline for my project, and I'm surprised by the difficulties in reproducing this paper.
